src/main/scala/com/lynbrookrobotics/sbt/Tasks.scala

```diff
@@ -32,7 +32,6 @@ object Tasks {
 
     List(
       s"roboRIO-${Keys.teamNumber.value}-FRC.local", // mDNS
-      s"roboRIO-${Keys.teamNumber.value}-FRC.lan", // mDNS
```

Comment:
Any reason this was dropped?

Reply:
Sorry, my mistake. I thought that address was never used, but after re-searching the manual:
`2. DNS (roboRIO-TEAM-FRC.lan where TEAM is your team number)`